### PR TITLE
Add new feature for Activity repetition.

### PR DESCRIPTION
**Pull request by idegtiarov**
- Add changes to the SequenceItem model
- Add suffix to the user_id in the consumer module
- Add changes to the mock and vpal engines.
- Add tests.

### CODE REVIEW

File: bridge_adaptivity/module/models.py [hunk 1]
```diff
@@ -109,6 +111,7 @@ class SequenceItem(models.Model):
     activity = models.ForeignKey('Activity', null=True, on_delete=models.CASCADE)
     position = models.PositiveIntegerField(default=1)
     score = models.FloatField(null=True, blank=True, help_text="Grade policy: 'p' (problem's current score).")
+    suffix = models.CharField(max_length=10, default='')
```
Comment (flying-pi) [inline bridge_adaptivity/module/models.py]: It is a too common name, please add some comment

File: bridge_adaptivity/bridge_lti/consumer.py [hunk 2]
```diff
@@ -71,6 +71,8 @@ def source_preview(request):
         # student flow
         sequence_item = SequenceItem.objects.get(id=sequence_item_id)
         activity = sequence_item.activity
+        if activity.repetition > 1:
```
Comment (flying-pi) [inline bridge_adaptivity/bridge_lti/consumer.py]: Can you move this logic into getters/setters of the model for localizing it?

File: bridge_adaptivity/module/models.py (hunk 1 above)
Comment (idegtiarov) [reply bridge_adaptivity/module/models.py]: done

File: bridge_adaptivity/bridge_lti/consumer.py (hunk 2 above)
Comment (idegtiarov) [reply bridge_adaptivity/bridge_lti/consumer.py]: done

**Pull request merged**